src/lib/ipsec/esp.lua

```diff
@@ -48,6 +48,8 @@ local PAYLOAD_OFFSET = ETHERNET_SIZE + IPV6_SIZE
 local ESP_NH = 50 -- https://tools.ietf.org/html/rfc4303#section-2
 local ESP_SIZE = esp:sizeof()
 local ESP_TAIL_SIZE = esp_tail:sizeof()
+local ESP_RESYNC_THRESHOLD = 4 -- Resync after this many consecutive failed decaps
+local ESP_RESYNC_ATTEMPTS = 6 -- Try this many increments of seq_high
```

Comment:
These should be configurable in `new` like spi etc. What could be sensible defaults?